pkg/assets/create/creater.go

```diff
@@ -3,13 +3,12 @@ package create
 import (
 	"context"
 	"fmt"
-	"io"
-	"os"
 	"sort"
 	"strings"
 	"time"
 
 	"github.com/ghodss/yaml"
+	"github.com/go-log/log"
```

Comment:
no new logging dep.  Project standard is glog and we'll switch to klog with kube.